examples/log-ingestion/docker-compose-dataprepper.yaml

```diff
@@ -0,0 +1,15 @@
+version: ‘3.7’
+services:
+  data-prepper:
+    image: opensearchproject/data-prepper
+    container_name: data-prepper
+    volumes:
+      - ./log_pipeline.yaml:/usr/share/data-prepper/pipelines/log_pipeline.yaml
+    ports:
+      - 2021:2021
+    networks:
+    - opensearch-net
+
+networks:
+  opensearch:
```

Comment:
Shouldn't this be `opensearch-net` to be aligned to the network listed for the data-prepper service?

Reply:
yes,